playwright-tests/tests/mentorship.page.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { expect } from '@playwright/test';
+
+import { test } from '@utils/fixtures';
+
+// MENT-004: Verify title and testimonial cards are displayed correctly
+test.describe('Mentorship Feedback Section', () => {
+  test('MENT-004: Verify title and cards are correctly displayed', async ({
+    mentorshipPage,
+    basePage,
+  }) => {
+    await basePage.navigateToPath('/mentorship');
+
+    await expect(mentorshipPage.testimonialsTitle).toBeVisible();
+    await expect(mentorshipPage.testimonialsTitle).toHaveText(
+      'What do participants think about our Mentorship Programme?',
+    );
```

Comment:
In this case, the `toHaveText()` assertion is redundant because the locator already contains the text. It is sufficient to simply verify that the element is visible on the page (line 13).